docs/installation.rst

```diff
@@ -4,153 +4,126 @@
 Installation
 ============
 
-|project_name| pyinstaller binary installation requires:
-
-* Linux: glibc >= 2.13 (ok for most supported Linux releases)
-* MacOS X: 10.10 (unknown whether it works for older releases)
-* FreeBSD: 10.2 (unknown whether it works for older releases)
-
-|project_name| non-binary installation requires:
-
-* Python_ >= 3.2.2
-* OpenSSL_ >= 1.0.0
-* libacl_ (that pulls in libattr_ also)
-* liblz4_
-* some python dependencies, see install_requires in setup.py
-
-General notes
--------------
-You need to do some platform specific preparation steps (to install libraries
-and tools) followed by the generic installation of |project_name| itself:
-
-Below, we describe different ways to install |project_name|.
-
-- **dist package** - easy and fast, needs a distribution and platform specific
-  binary package (for your Linux/*BSD/OS X/... distribution).
-- **pyinstaller binary** - easy and fast, we provide a ready-to-use binary file
-  that just works on the supported platforms
-- **pypi** - installing a source package from pypi needs more installation steps
-  and will need a compiler, development headers, etc..
+There are different ways to install |project_name|:
+
+- **distribution package** - easy and fast if a package is available for your
+  Linux/BSD distribution.
+- **PyInstaller binary** - easy and fast, we provide a ready-to-use binary file
+  that comes bundled with all dependencies.
+- **pip** - installing a source package with pip needs more installation steps
+  and requires all dependencies with development headers and a compiler.
 - **git** - for developers and power users who want to have the latest code or
   use revision control (each release is tagged).
 
-**Python 3**: Even though this is not the default Python version on many systems,
-it is usually available as an optional install.
 
-Virtualenv_ can be used to build and install |project_name| without affecting
-the system Python or requiring root access.
+Installation (Distribution Package)
+-----------------------------------
 
-Important:
-If you install into a virtual environment, you need to **activate**
-the virtual env first (``source borg-env/bin/activate``).
-Alternatively, directly run ``borg-env/bin/borg`` (or symlink that into some
-directory that is in your PATH so you can just run ``borg``).
-Using a virtual environment is optional, but recommended except for the most
-simple use cases.
+Some Linux and BSD distributions might offer a ready-to-use ``borgbackup``
+package which can be installed with the package manager.  As |project_name| is
+still a young project, such a package might be not available for your system
+yet. Please ask package maintainers to build a package or, if you can package /
+submit it yourself, please help us with that!
 
-The llfuse_ python package is also required if you wish to mount an
-archive as a FUSE filesystem. Only FUSE >= 2.8.0 can support llfuse.
+* On **Arch Linux**, there is a package available in the AUR_.
```

Comment:
The Arch AUR is not part of the official repos, so I think it's good to have this link somewhere. And at this stage users might not expect to find borg through their package manager and not even try.